shared/style/switches.css

```diff
@@ -80,26 +80,26 @@ label.pack-radio.danger input ~ span:after {
  * ---------------------------------- */
 
 label.pack-switch input ~ span:after {
-  content: 'Off';
+  content: 'OFF';
```

Comment:
Please correct me if I'm wrong, but it seems to me, that ``content: `` value will be shown right to the user without any localization. If so - you most probably can't do this, due to Mozilla requirements to have strings localized.